quill-async-postgres/src/test/scala/io/getquill/context/async/postgres/ArrayExtraAsyncEncodingSpec.scala

```diff
@@ -0,0 +1,56 @@
+package io.getquill.context.async.postgres
+
+import java.time.{ ZonedDateTime }
+
+import io.getquill.context.sql.encoding.ArrayEncodingExtraSpec
+import org.joda.time.{ DateTime => JodaDateTime }
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class ArrayExtraAsyncEncodingSpec extends ArrayEncodingExtraSpec {
```

Comment:
that's correct, but I was asking to add just one more unit test to existing suite as in https://github.com/getquill/quill/blob/master/quill-async-postgres/src/test/scala/io/getquill/context/async/postgres/ArrayAsyncEncodingSpec.scala#L23-L31